bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -689,12 +699,21 @@ public void cancelExecution(String executionId) {
     }
 
     public void executeActionGroup(String id) {
+        if (ThingStatus.OFFLINE.equals(thing.getStatus()) && !reLogin()) {
```

Comment:
Because `ThingStatus` is an enum so you can use == in comparisons:

```suggestion
        if (ThingStatus.OFFLINE == thing.getStatus() && !reLogin()) {
```

Reply:
changed